media-libs/x265/x265-3.4.ebuild

```diff
@@ -6,15 +6,16 @@ EAPI=7
 inherit cmake multilib-minimal multilib multibuild flag-o-matic
 
 if [[ ${PV} = 9999* ]]; then
-	inherit mercurial
-	EHG_REPO_URI="https://bitbucket.org/multicoreware/x265"
+	inherit git-r3
+	EGIT_REPO_URI="https://bitbucket.org/multicoreware/x265_git/"
+	S=${WORKDIR}/${P}/source
 else
-	SRC_URI="https://bitbucket.org/multicoreware/x265/downloads/${PN}_${PV}.tar.gz"
+	SRC_URI="https://bitbucket.org/multicoreware/x265_git/get/${PV}.tar.gz -> ${P}.tar.gz"
 	KEYWORDS="amd64 arm arm64 hppa ~ia64 ppc ppc64 x86"
 fi
 
 DESCRIPTION="Library for encoding video streams into the H.265/HEVC format"
-HOMEPAGE="http://x265.org/ https://bitbucket.org/multicoreware/x265/wiki/Home"
+HOMEPAGE="http://x265.org/ https://bitbucket.org/multicoreware/x265_git/wiki/Home"
```

Comment:
HOMEPAGE should be http://x265.org/ and https://www.videolan.org/developers/x265.html

Reply:
http://x265.org/ and https://www.videolan.org/developers/x265.html and http://hg.videolan.org/x265 are not uptodate.
https://x265.org expired certificate
https://bitbucket.org/multicoreware/x265_git/ is the only way to go at the moment.
Please stabilize media-libs/x265-3.5 and remove 3.4!